src/main/java/backend/model/payment/SponsoringInvoice.kt

```diff
@@ -118,7 +118,9 @@ class SponsoringInvoice : Invoice {
     }
 
     private fun Sponsoring.toEmailListing(): String {
-        return "<b>Team-ID</b> ${this.team?.id} <b>Teamname</b> ${this.team?.name} <b>Status</b> ${this.status} <b>Betrag pro km</b> ${this.amountPerKm.display()} <b>Limit</b> ${this.limit.display()} <b>Gereiste KM</b> ${this.team?.getCurrentDistance()} <b>Spendenversprechen</b> ${this.billableAmount().display()}"
+        // TODO: implement with teams
```

Comment:
I'm honestly not sure how this should be correctly implemented.

We need to check who this is sent to? The sponsor? Each team?

Based on this we either list all teams here (if sent to sponsor) or add a team to the function parameters and only show this team's name+id+distance+km+billable amount. And not all other teams they're not interested in.